### PR TITLE
[6.5][.NET] Removed redundant widget description.

### PR DESCRIPTION
**Pull request by mihashco**

### CODE REVIEW

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco , I have reviewed and added my feedback comments. Kindly check and incorporate the changes.

File: docs/application/dotnet/guides/applications/uiapplication/widget-app.md [hunk 1]
```diff
@@ -1,47 +1,105 @@
...
 
-You can create widget applications, which are Tizen .NET applications shown on the home screen. They are launched by the home screen when the user selects them in the **Add Widget** menu.
+The widget applications are commonly used in applications like home screen or lock screen. The .NET Tizen API provides two class that allows you to implement widget applications:
 
-The main widget application features include:
+1. [NUIWidgetApplication](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html)
+2. [WidgetApplication](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.WidgetApplication.html)
 
-- Creating widget applications
+An instance of the widget is managed by a [Widget Viewer](./) application. Widget process is created when a widget application is added to the widget viewer.
```
[the remark this answers is not in the document]
Comment (mihashco) [reply docs/application/dotnet/guides/applications/uiapplication/widget-app.md:8]: This file is introduced in #1505

File: docs/application/dotnet/guides/applications/uiapplication/widget-app.md [hunk 2]
```diff
@@ -20,22 +29,71 @@ The main widget application features include:
...
+        <table>
+            <tbody>
+                <tr>
+                    <p>The <a href="/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.WidgetApplication.html">Tizen.Applications.WidgetApplication</a> class provides the <code>WidgetApplication(IDictionary&lt; Type, string &gt; typeInfo)</code> constructor, which allows a widget application to have multiple widget classes. The widget applications with multiple classes can make diverse class instances whenever widget viewer applications, such as the home screen and the lock screen, request for a widget instance.</p>
+                    <p>The widget instance has its own life-cycle similar to the widget application. However, the widget instance is only an object shown by the widget viewer applications. Many widget instances can be running on the same widget application process.</p>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
```
[the remark this answers is not in the document]
Comment (mihashco) [reply docs/application/dotnet/guides/applications/uiapplication/widget-app.md:98]: I'm not sure how to do that. Markdown file don't create those spaces.

File: docs/application/dotnet/guides/applications/uiapplication/widget-app.md [hunk 3]
```diff
@@ -1,47 +1,105 @@
-# Widget Application
+# Widget application
```
[the remark this answers is not in the document]
Comment (sagarkedari) [reply docs/application/dotnet/guides/applications/uiapplication/widget-app.md:1]: I still issue with heading format in this topic. Following headings are still in title case. Please change to sentence case.

- Widget Instance States and Events
- Data Sharing Between the widget application and other applications

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco , I have only following comment to be addressed yet. Please check and incorporate the changes.

**Review by sagarkedari — APPROVED** [review]
@annie-samsung, This PR is approved from my end.

**Pull request merged**